File: Logica-Programacao/Javascript/Faccat/ex017.js
let avaliacao1, avaliacao2, media;

avaliacao1 = parseFloat(prompt("Nota da avaliação 1: "));
avaliacao2 = parseFloat(prompt("Nota da avaliação 2: "));

media = (avaliacao1 + avaliacao2) / 2;

alert("Média: " + media);

if (media >= 6) {
    alert("Aprovado!");
} else {
    alert("Reprovado!")
}

File: Java/Logica-Programacao/src/faccat/Ex017.java
package faccat;

import java.util.Scanner;

public class Ex017 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite a 1° nota do aluno: ");
        float nota1 = sc.nextFloat();
        System.out.print("Digite a 2° nota do aluno: ");
        float nota2 = sc.nextFloat();
        sc.close();

        float media = (nota1 + nota2) / 2;

        if (media >= 6) {
            System.out.printf("Com a média de %.1f o aluno é aprovado.\n", media);
        } else System.out.printf("Com a média de %.1f o aluno é reprovado.\n", media);
    }
}